devito/ir/support/properties.py

```diff
@@ -16,11 +16,17 @@ def __init__(self, name, val=None):
 PARALLEL = Property('parallel')
 """A fully parallel Dimension."""
 
+PARALLEL_INDEP = Property('parallel=')
+"""
+A fully parallel Dimension; parallelism is due to all dependences having
+distance 0 (i.e., distance vector is '='). This is stronger than PARALLEL.
+"""
+
 PARALLEL_IF_ATOMIC = Property('parallel_if_atomic')
-"""A parallel Dimension with local reductions."""
+"""A parallel Dimension with local reductions. This is weaker than PARALLEL."""
```

Comment:
Maybe a docstring can be added, with notes, at class level denoting the priority/strength of properties.
Like PARALLEL_INDEP>PARALLEL>PARALLEL_IF_ATOMIC